code/modules/antagonists/clock_cult/mobs/eminence.dm

```diff
@@ -47,6 +47,9 @@
 	if(!.)
 		A.eminence_act(src)
 
+/mob/living/simple_animal/eminence/attack_animal(mob/living/simple_animal/M)
+	return
+
```

Comment:
`attack_animal` is a defense proc; overriding eminence changes how eminence *reacts to an attack by simple animals*. If you want to prevent `attack_animal` being called by eminence (and therefore the nuzzling at `/mob/living/attack_animal` and `/obj/attack_animal`), you can override `UnarmedAttack`.

Reply:
:+1: